tests/rgw/test_s3.py

```diff
@@ -569,13 +571,17 @@ def _rgw_lc_debug(cluster: Ceph, add: bool = True) -> None:
     for daemon in rgw_daemons:
         if add:
             command = f"ceph config set {daemon} rgw_lc_debug_interval 10"
+            command_1 = f"ceph config set {daemon} rgw_sts_key abcdefghijklmnop"
+            command_2 = f"ceph config set {daemon} rgw_s3_auth_use_sts true"
         else:
             command = f"ceph config rm {daemon} rgw_lc_debug_interval"
 
         node.exec_command(sudo=True, cmd=command)
+        node.exec_command(sudo=True, cmd=command_1)
```

Comment:
@viduship this can cause `command_1 variable not defined` error for else part right!
could you please check..

Reply:
done